doc/api/axes_api.rst

```diff
@@ -111,9 +111,10 @@ Statistics
 
    Axes.boxplot
    Axes.violinplot
+   Axes.ecdf
```

Comment:
```suggestion
   Axes.ecdf
   Axes.violinplot
```
? (As you ordered the two remaining ones...)

Reply:
sure